nuxeo-features/nuxeo-automation/nuxeo-automation-jsf/src/main/java/org/nuxeo/ecm/automation/jsf/operations/DownloadFile.java

```diff
@@ -40,15 +32,22 @@
 import org.nuxeo.ecm.platform.ui.web.util.BaseURL;
 import org.nuxeo.runtime.api.Framework;
 
+import javax.faces.context.ExternalContext;
+import javax.faces.context.FacesContext;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Collections;
+
```

Comment:
Are you using the Nuxeo formatter? Usually the `org`.* imports come after the `javax.*` ones.